TEMPORARY-topiary-tutorial-part-1/tree-sitter-yolo/topiary-yolo.ncl

```diff
@@ -0,0 +1,8 @@
+{
+  languages = {
+    yolo = {
+      extensions = ["yolo"],
+      grammar.source.path = "/home/yago/Pro/Tweag/blog-posts/topiary-tutorial/tree-sitter-yolo/yolo.so",
```

Comment:
Maybe it would be worth putting a comment in here or a dummy path (or both), so people following along can change this appropriately.